src/Options/Background.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Onliner\ImgProxy\Options;
+
+use Onliner\ImgProxy\Support\Color;
+
+final class Background extends Option
+{
+    /**
+     * @var Color
+     */
+    private $color;
+
+    public function __construct(string $color)
+    {
+        $this->color = new Color($color);
+    }
```

Comment:
я бы передавал явно класс:

```suggestion
    public function __construct(Color $color)
    {
        $this->color = $color;
    }
```

Reply:
я думаю это только усложнит создание опций, а плюсов этого решения не вижу.
под капотом я использую класс Color только для того, чтобы не повторять валидацию.

Reply:
все же да, более типизированные аргументы здесь только усложнят создание опций